package.json

```diff
@@ -140,7 +141,7 @@
 		"test-pmd-cataloger": "./gradlew pmd-cataloger:test pmd-cataloger:jacocoTestCoverageVerification",
 		"test-sfge": "./gradlew sfge:test sfge:jacocoTestCoverageVerification",
 		"test-sfge-quiet": "cross-env SFGE_LOGGING=false ./gradlew sfge:test sfge:jacocoTestCoverageVerification",
-		"test-typescript": "nyc mocha --timeout 10000 --retries 5 \"./test/**/*.test.ts\"",
+		"test-typescript": "nyc mocha --timeout 60000 \"./test/**/*.test.ts\"",
```

Comment:
The E2E tests are slower now. Increased the timeout to 60 seconds to make absolutely sure Windows can get through them, and removed the retries because running a failing test 6 times is unacceptably slow.